docs/nddata/index.rst

```diff
@@ -24,7 +24,8 @@ datasets in astropy through:
 .. warning::
 
   `~astropy.nddata` has changed significantly in astropy 1.0. See the section
-  :ref:`nddata_transition` for more information.
+  :ref:`nddata_transition` for more information. Some more changes were done in
+  astropy x.xx, see more details in :ref:`nddata_transition2`.
```

Comment:
I think we could take out this warning box altogether...it has been (almost?) a year since 1.0 was released.